commit b1c1b44ea2c9da008ee7d23864b7f28026ff5a3a (refs/changes/60/2360/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-08-09T21:15:34+00:00 (2 years, 2 months ago)
    
    MANTA-3369 update cueball to 2.2.9 to help debug long requests
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/package.json b/package.json
index 160f8d8..4c9e8a7 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "2.2.8",
+        "cueball": "2.2.9",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
